app/javascript/components/Checkout/PaymentForm.tsx

```diff
@@ -601,11 +601,13 @@ const CustomerDetails = () => {
       {state.products.length === 1 && state.products[0]?.canGift && !state.products[0]?.payInInstallments ? (
         <GiftForm isMembership={state.products[0]?.nativeType === "membership"} />
       ) : null}
-      <div>
-        <Button color="primary" onClick={() => dispatch({ type: "offer" })} disabled={isSubmitDisabled(state)}>
-          {payLabel}
-        </Button>
-      </div>
+      {state.paymentMethod !== "paypal" ? (
```

Comment:
Since `<CustomerDetails />` has been moved out of `<CreditCard />` now to avoid conditional rendering, we need to avoid displaying the `Pay` button here when PayPal is selected because PayPal has its own buttons.